docs/en/integrations/data-ingestion/kafka/index.md

```diff
@@ -821,7 +870,8 @@ ClickHouse Kafka Connect reports the following metrics:
 ### Limitations
 - Deletes are not supported.
 - Batch size is inherited from the Kafka Consumer properties.
-- When using KeeperMap for exactly-once and the offset is changed or rewound, you need to delete the content from KeeperMap for that specific topic.
+- When using KeeperMap for exactly-once and the offset is changed or rewound, you need to delete the content from KeeperMap for that specific topic. (See troubleshooting guide below for more details)
+
 
 ### Troubleshooting
 #### "I would like to adjust the batch size for the sink connector"
```

Comment:
I am guessing that this is still being worked on; please ping me when it is ready to be merged.